Resources/Private/Templates/Frontend/Search/Form.html

```diff
@@ -0,0 +1,9 @@
+<html xmlns="http://www.w3.org/1999/xhtml" lang="en"
+	  xmlns:f="http://typo3.org/ns/TYPO3/Fluid/ViewHelpers"
+	  xmlns:s="http://typo3.org/ns/ApacheSolrForTypo3/Solr/ViewHelpers/Frontend/">
+
+<f:layout name="Frontend/Main"/>
+
+<f:section name="main">
+	<f:render partial="Frontend/Search/Form" section="Form" arguments="{search:search, additionalFilters:additionalFilters}" />
```

Comment:
I think this will not work by "Use custom Fluid Templates", if user sets
`partialRootPaths.10 = EXT:own_extension/Resources/Private/Partials/Frontend/`
I prefer `<f:render partial="Search/Form" section="Form" arguments="{search:search, additionalFilters:additionalFilters}" />`
and using `.../Frontend/` everywhere in TypoScript configs